packages/website/src/components/homepage/homepage.tsx

```diff
@@ -0,0 +1,219 @@
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import { Links } from "@site/src/constants";
+import { DataValidationHeroIllustration } from "@site/src/pages/data-validation";
+import { OpenAPI3HeroIllustration } from "@site/src/pages/openapi";
+import { Button } from "../button/button";
+import { CodeBlock } from "../code-block/code-block";
+import { FeatureList } from "../feature-list/feature-list";
+import { HeroTabs } from "../hero-tabs/hero-tabs";
+import { LearnMoreCard } from "../learn-more-card/learn-more-card";
+import { LightDarkImg } from "../light-dark-img/light-dark-img";
+import { Section } from "../section/section";
+import { SectionedLayout } from "../sectioned-layout/sectioned-layout";
+import { DescriptionText, PrimaryText } from "../text/text";
+import style from "./homepage.module.css";
+import { OverviewIllustration } from "./overview-illustration/overview-illustration";
+
+export const HomeContent = () => {
+  return (
+    <>
+      <Hero />
+      <div className={style["hero-divider"]}></div>
+      <SectionedLayout>
+        <Overview />
+        <OpenAPISection />
+        <DataValidationSection />
+        <EditorSection />
+        <ExtensibilitySection />
+        <Closing />
+      </SectionedLayout>
+    </>
+  );
+};
+
+const Hero = () => {
+  return (
+    <>
+      <div className={style["hero-container"]}>
+        <div className={style["hero-content"]}>
+          <h1 className={style["hero-title"]}>TypeSpec</h1>
+          <div className={style["hero-subtitle"]}>Describe APIs at scale</div>
+          <DescriptionText size="large" className={style["hero-description"]}>
+            Describe your data up front and generate schemas, API specifications, client / server
+            code, docs, and more.
+          </DescriptionText>
+          <div className={style["hero-buttons"]}>
+            <Button as="a" appearance="primary" href={useBaseUrl(Links.docs)}>
+              Get Started
+            </Button>
+            <Button as="a" appearance="outline" href={useBaseUrl(Links.playground)}>
+              Try it out
+            </Button>
+          </div>
+        </div>
+        <div className={style["hero-demo"]}>
+          <HeroTabs
+            tabs={[
+              {
+                value: "OpenAPI",
+                content: (
+                  <DotLottiePlayer src={useBaseUrl(`/lottie/openapi-hero.lottie`)} autoplay loop />
```

Comment:
File size is much better but now the decoding does take a little bit of time which happens everytime you switch tabs which is not optimal at all.